python/ray/autoscaler/_private/kuberay/node_provider.py

```diff
@@ -211,6 +229,12 @@ def _worker_group_replicas(raycluster: Dict[str, Any], group_index: int):
     return raycluster["spec"]["workerGroupSpecs"][group_index].get("replicas", 1)
 
 
+def _worker_group_num_of_hosts(raycluster: Dict[str, Any], group_index: int):
```

Comment:
Where is this function called?

Reply:
This function isn't used anywhere else in the PR currently, I just thought it could be useful to add it to match the existing getter functions, but for the new `NumOfHosts` field.

Reply:
Nevermind, this function is called [here](https://github.com/ray-project/ray/pull/43470/files#diff-a3d493ad71c57ecd60da4d4824f2ce332da9992d0807634c0d88098a008eed98R473) to determine the correct number of target replicas, since it previously did not account for `NumOfHosts`.